web-sdk.js

```diff
@@ -1876,8 +1871,14 @@
                 i += "; expires=" + o.toGMTString();
             }
             "path" in n && (i += "; path=" + n.path);
+            "domain" in n && (i += "; domain=" + n.domain);
```

Comment:
What does this addition address?

Reply:
It was removed before my me because it was causing issues with local, but since it's not being used anymore shouldn't be a problem